integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -130,8 +130,20 @@ spec:
           - configMapRef:
               name: {{ $fullName }}-config
 {{- $probePort := $isHaEmbedded | ternary "embedded" "rpc" }}
-{{ include "alluxio.master.readinessProbe" (dict "port" $probePort) | indent 10 }}
-{{ include "alluxio.master.livenessProbe" (dict "port" $probePort) | indent 10 }}
+          readinessProbe:
+            tcpSocket:
+              port: {{ $probePort }}
+            initialDelaySeconds: {{ .Values.master.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.master.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.master.readinessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.master.readinessProbe.failureThreshold }}
+          livenessProbe:
+            tcpSocket:
+              port: {{ $probePort }}
+            initialDelaySeconds: {{ .Values.master.livenessProbe.initialDelaySeconds }}
```

Comment:
Same rationale, need a null check like `{{- if .Values.master.livenessProbe }}`?

Reply:
With default, there is no need for null check.